install/error-handling.sh

```diff
@@ -1,10 +1,36 @@
 echo "${_group}Setting up error handling ..."
 
+export SENTRY_DSN='https://19555c489ded4769978daae92f2346ca@self-hosted.getsentry.net/3'
+export SENTRY_ORG=sentry
```

Comment:
Can we use `self-hosted`? Makes it easier to distinguish between ours and SaaS.

<img width="633" alt="Screen Shot 2022-09-07 at 2 48 51 PM" src="https://user-images.githubusercontent.com/134455/188955045-dcafc845-9514-445f-80b7-39863440ac88.png">

Reply:
It seems that I would have to re-create the sentry instance to change this. I can do that if you want.

Reply:
Done, changed on both sides, forgot there was a setting for this 😓